.github/workflows/helm-deploy-testkube-charts-stage.yaml

```diff
@@ -0,0 +1,169 @@
+name: Releasing Testkube Helm charts to Stage k8s Cluster.
+
+on:
+  repository_dispatch:
+    types: [trigger-workflow]
+
+env:
+  PROJECT_ID: ${{ secrets.GKE_PROJECT }}
+  GKE_CLUSTER_NAME_STAGE: ${{ secrets.GKE_CLUSTER_NAME_STAGE }} # Add your cluster name here.
+  GKE_ZONE_STAGE: ${{ secrets.GKE_ZONE_STAGE }} # Add your cluster zone here.
+  DEPLOYMENT_NAME: testkube # Add your deployment name here.
+  POSTMAN_TEST_FILE_NANE: TestKube-Sanity.postman_collection.json
+  POSTMAN_TEST_FILE_PATH: test/postman
+
+jobs:
+  deploy-to-testkube-stage-gke:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.client_payload.ref }}
+      - run: echo ${{ github.event.client_payload.image_tag }}
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      # Setup gcloud CLI
+      - uses: google-github-actions/setup-gcloud@94337306dda8180d967a56932ceb4ddcf01edae7
+        with:
+          service_account_key: ${{ secrets.GKE_SA_KEY }}
+          project_id: ${{ secrets.GKE_PROJECT }}
+
+      # Configure Docker to use the gcloud command-line tool as a credential
+      # helper for authentication
+      - run: |-
+          gcloud --quiet auth configure-docker
+
+      # Get the GKE credentials so we can deploy to the cluster
+      - uses: google-github-actions/get-gke-credentials@fb08709ba27618c31c09e014e1d8364b02e5042e
+        with:
+          cluster_name: ${{ env.GKE_CLUSTER_NAME_STAGE }}
+          location: ${{ env.GKE_ZONE_STAGE }}
+          credentials: ${{ secrets.GKE_SA_KEY }}
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Installing repositories
+        run: |
+          helm repo add helm-charts https://kubeshop.github.io/helm-charts
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+
+      # Deploy the Docker image to the GKE cluster
+      - name: Deploy
+        run: |-
+          helm upgrade --install --atomic --timeout 180s testkube helm-charts/testkube --namespace testkube --create-namespace --set testkube-api.image.tag=${{ github.event.client_payload.image_tag }} --values ./charts/testkube/values-demo.yaml
```

Comment:
Values for staging are needed + ingress DNS host

Reply:
`integration.testkube.io`? `staging.testkube.io` ?

Reply:
As per our discussion, a new issue will be created for Ingress/DNS setup.